### DISCUSSION TITLE
v61 box - need help

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp32_8ch_motor_shield#70] [General]
Hi, 
I have no time to make the new 3D box for the v61. 
Can anybody make the job ?
[3D_61-451_obj.zip](https://github.com/user-attachments/files/17791762/3D_61-451_obj.zip)
[3D_61-451_step.zip](https://github.com/user-attachments/files/17791763/3D_61-451_step.zip)
[DXF_61-451_2024-11-17_AutoCAD2007.zip](https://github.com/user-attachments/files/17791797/DXF_61-451_2024-11-17_AutoCAD2007.zip)

**Comment by MrAges** [comment]
hey,
i made a simple case. should fit but not printed yet. i´m open for feedback

[motor_shield_case.zip](https://github.com/user-attachments/files/18344891/motor_shield_case.zip)

![image](https://github.com/user-attachments/assets/a7d9b9d6-6687-4b7a-81df-79a352eff693)

regards

**Reply by nliaudat** [reply]
Many thanks. 
I've uploaded to https://github.com/nliaudat/esp32_8ch_motor_shield/tree/main/box/v60

**Reply by derpeter** [reply]
@MrAges i will print the case in the next days and will add some "ears" with holes to mount the case. I will probably just do this in thinkercat as my 3d modeling skills are very basic so i guess the result will work for me but be not as nice as if someone with more skills do it for uploading it here.

**Reply by MrAges** [reply]
@derpeter if you send a sketch i can add it to my model - just few minutes work.
